data-sources/browser-signals/config.yml

```
@@ -0,0 +1,9 @@
+id: browser-signals
+displayName: Browser Signals
+description: |
+  Comprehensive Browser dashboard providing actionable insights, including CWV, Throughput, Errors & Latency. 
+install:
+  primary:
+    link:
+      url: https://docs.newrelic.com/docs/new-relic-solutions/best-practices-guides/full-stack-observability/browser-monitoring-best-practices-guide/
+  icon: logo.svg
```

The formatting is a little messed up here
```suggestion
icon: logo.svg
```